gatsby-config.js

```diff
@@ -24,7 +24,12 @@ module.exports = {
       },
     },
     "gatsby-plugin-react-helmet",
-    "gatsby-plugin-sass",
+    {
+      resolve: "gatsby-plugin-sass",
+      options: {
+        additionalData: `@import "${__dirname}/src/styles/__colors.scss";`,
```

Comment:
why chose this config instead of what we did for fonts and just include them in the top level